include/Capabilities/Capabilities.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Capabilities;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * A class allowing to determine if a user can translate content.
```

Comment:
```suggestion
 * A class allowing to alias `manage_translations` and `manage_options`.
```

Reply:
What do you think of this instead? I don't think we should be too specific.

```
A class allowing to map Polylang's custom user capabilities to WP's native ones.
```

Reply:
Done in [c32def2](https://github.com/polylang/polylang/pull/1727/commits/c32def2e0eb321be1980050e98e1c6b392ff088b).